sources/packages/api/src/services/application/application.service.e2e-spec.ts

```diff
@@ -181,7 +181,6 @@ describe("ApplicationService", () => {
         expect(msfaaNumberService.createMSFAANumber).toHaveBeenCalled();
       } finally {
         await applicationRepository.remove(testCompletedApplication);
-        await applicationRepository.remove(testApplication);
```

Comment:
Removed it, as the related msfaa number has been removed during the testCompletedApplication removed in line 183